src/main/java/be/acara/frontend/service/UserServiceImpl.java

```diff
@@ -60,6 +60,32 @@ public void editUser(UserModel user) {
         editFrontEndUser(user);
         jwtTokenService.remove(user.getUsername());
     }
+
+    @Override
+    public boolean doesUserLikeThisEvent(Long id) {
+        return userFeignClient.checkIfUserLikesThisEvent(id);
+    }
+
+    @Override
+    public void likeOrDislikeEvent(Long id) {
+        if(doesUserLikeThisEvent(id)){
+            likeEvent(id);
+        }
+        else{
+            dislikeEvent(id);
+        }
+    }
+
+
+    @Override
+    public void likeEvent(Long id) {
+        userFeignClient.likeEvent(id);
+    }
+
+    @Override
+    public void dislikeEvent(Long id) {
+        userFeignClient.dislikeEvent(id);
```

Comment:
This id seems to map to `/api/users/{id}/events`, but you are giving it an event id. How does this work?
Remember: For API endpoint `/api/users/{id}/events`; `{id}` implies `{userId}`. A more correct API endpoint would be: `/api/users/{userId}/likes/{eventId}`